conf/modules.config

```diff
@@ -68,6 +68,16 @@ if (!params.skip_trimming) {
 }
 
     // PREPARE GENOME
+    withName: CLEAN_FASTA {
+        ext.args2 = '\'/>/{ gsub(\$2, "",\$2);gsub(" ", "") };{print}\''
+        publishDir = [
```

Comment:
Is it important that this is placed in the outdir?

Reply:
not really
I'm not sure what the guidelines are. Do we want the users to have access to all the reference files?